client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -77,12 +81,17 @@ class StatModuleFollowers extends Component {
 		const summaryPageSlug = siteSlug || '';
 		// email-followers is no longer available, so fallback to the new subscribers URL.
 		// Old, non-functional path: '/people/email-followers/' + summaryPageSlug.
-		let summaryPageLink = '/people/subscribers/' + summaryPageSlug;
+		// If the site is Atomic or Jetpack self-hosted, it links to Jetpack cloud.
+		let summaryPageLink =
+			isAtomic || isJetpack
+				? `https://cloud.jetpack.com/subscribers/${ summaryPageSlug }`
+				: `/people/subscribers/${ summaryPageSlug }/${ isAtomic }/${ isJetpack }`;
```

Comment:
This looks a bit weird, boolean values in the URL 🤔

Reply:
Ah noticed this broke the Calypso Stats link, I created a followup PR: https://github.com/Automattic/wp-calypso/pull/89311